src/components/Modal/UpdateProposalStatusModal/UpdateProposalStatusModal.tsx

```diff
@@ -76,43 +62,85 @@ export function UpdateProposalStatusModal({
   ...props
 }: Props) {
   const t = useFormatMessage()
-  const isOneTimePaymentProposalTier = GrantTier.isOneTimePaymentTier(proposal?.configuration.tier)
   const queryClient = useQueryClient()
+  const { vesting_addresses } = proposal || {}
+  const DEFAULT_VALUES = useMemo(
+    () => (vesting_addresses && vesting_addresses.length > 0 ? vesting_addresses : ['']),
+    [vesting_addresses]
+  )
 
   const onClose = () => {
     setError('')
+    setValue('vestingAddresses', DEFAULT_VALUES)
     props.onClose()
   }
 
-  const {
-    handleSubmit,
-    formState: { isSubmitting, errors },
-    control,
-  } = useForm<UpdateProposalState>({ defaultValues: initialUpdateProposalState, mode: 'onTouched' })
+  const { handleSubmit, control, setValue } = useForm<UpdateProposalState>({
+    defaultValues: { vestingAddresses: DEFAULT_VALUES },
```

Comment:
Maybe this should be `vestingAddressesDefaultValue` or move the object to the `defaultValues` variable.